.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = tab
```

Comment:
My eyes :) 

![tabs-vs-spaces](https://media.giphy.com/media/l0IylSajlbPRFxH8Y/giphy.gif)

Reply:
hehe, ja, hva skal vi si. Jeg er vel for spaces altså, men det er tabs fra 2006. Ta det med gamlekara.